Formula/q/qscintilla2.rb

```diff
@@ -29,11 +29,11 @@ class Qscintilla2 < Formula
 
   depends_on "pyqt-builder" => :build
   depends_on "pyqt"
-  depends_on "python@3.13"
+  depends_on "python@3.14"
```

Comment:
Revision is not updated, not sure this is intended or not. Anyway we can do it later.